package.json

```diff
@@ -1,19 +1,35 @@
 {
   "name": "polymer-project-builder",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "Custom Polymer project builder",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --coverage",
```

Comment:
Does jest check coverage and exit with `1` when is not covered enough?

Reply:
Yes